### ISSUE TITLE
is it possible to use `mask` in template matching or keypoint matching?

### ISSUE THREAD

**Issue by spectereye** [AirtestProject/Airtest#518]
possible in opencv but seems not possible in airtest? could you please add it so we can match content of "circle" - currently only rectangular?

**Comment by Meteorix** [comment]
Good point. When we were writing airtest, opencv did not support input circle at that time. We will look into the latest version.




    发自网易邮箱大师


On 09/04/2019 22:08, spectereye wrote: possible in opencv but seems not possible in airtest? could you please add it so we can match content of "circle" - currently only rectangular?

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/AirtestProject/Airtest/issues/518?email_source=notifications\u0026email_token=AA5BAXVRHPYDGFCYQ4RQU2TQH66PFA5CNFSM4ITSMW52YY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4HJILPOQ",
"url": "https://github.com/AirtestProject/Airtest/issues/518?email_source=notifications\u0026email_token=AA5BAXVRHPYDGFCYQ4RQU2TQH66PFA5CNFSM4ITSMW52YY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4HJILPOQ",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by spectereye** [comment]
Greeeeat! again, thanks for your great work and looking forward for this new feature!
